RecoLocalTracker/SiPixelClusterizer/BuildFile.xml

```diff
@@ -1,7 +0,0 @@
-<use name="cuda"/>
-<use name="CalibTracker/SiPixelESProducers"/>
-<use name="FWCore/Utilities"/>
```

Comment:
Shouldn't this get propagated down to "plugin", then?

Reply:
> Thanks, you are right. Actually, most of them are already there, except for `FWCore/Utilities`. But I'm not sure if it's worth it to update this PR now (if there are no other issues). It has many signatures already. Almost everything depends on `FWCore/Utilities`, so maybe it's fine if `RecoLocalTracker/SiPixelClusterizer/plugins` inherits this dependency indirectly for now?

Agree: I'm going to sign this one
OTOH, I think that a quick additional PR could be made to restore the `FWCore/Utilities` where noticed missing: it will be fully orthogonal to this one, and it won't even need to wait for this one being actually merged